server.js

```diff
@@ -846,6 +604,7 @@ function run(configuration, configuredmailer) {
                 log.info("Login successful: " + profile.username);
                 done(null, profile);
             });
+            prChecker = new PRChecker(config, log, store, GH, mailer);
```

Comment:
`prChecker` should be instantiated outside `passport.use` as people aren't necessarily logged in the repo manager.

Reply:
thanks, that was a major bug indeed!

(too bad the test suite didn't catch this)